app/helpers/hiring_staff/job_creation_helper.rb

```diff
@@ -6,12 +6,20 @@ def current_step
   def steps_to_display
```

Comment:
This commit is called 'WIP: Apply classes' and not all of its changes are transparent. Can you make a more informative commit message?